angular15-vue3/e2e/checkAngular15Vue3App.cy.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  baseSelectors,
+  selectors,
+} from './../../cypress/common/selectors';
+import { Constants } from '../../cypress/fixtures/constants';
+import { BaseMethods } from '../../cypress/common/base';
+
+const basePage: BaseMethods = new BaseMethods();
+
+describe('Angular 15 and Vue 3 microfrontends', () => {
+    beforeEach(() => {
```

Comment:
Please add a `context` wrapper to test

It will be looks like:

```
describe('Angular 15 and Vue 3 microfrontends', () => {
    context("Check Angular 15 and Vue 3 elements", () => {
            beforeEach(() => {
                ...
             })
     })
  })
  ```

Reply:
resolved